track_info.c

```diff
@@ -110,6 +111,10 @@ void track_info_set_comments(struct track_info *ti, struct keyval *comments) {
 		ti->rg_album_gain = (r128_album_gain / 256.0) + 5;
 	}
 
+	if (comments_get_signed_int(comments, "output_gain", &output_gain) != -1) {
+		ti->output_gain = (output_gain / 256.0);
+	}
```

Comment:
Well, since the `output_gain` from the header is stored in the `comments`, this can't be moved to `track_info_new` because they aren't initialized there yet.

Reply:
I meant this code will only trigger for opus files, so `ti->output_gain` should be pre-initialized for the default case.

Reply:
Ah, that's what you meant. I completely missed that. I printed the value in the set_comment method before setting it (just to make sure it wasn't garbage) and it was always zero, so I assumed that the new structs were either `calloc`ed or `memset`. Now that you point it out, the existence of a `_new` method really does speak against that argument.
Fixed in 72bb979.